aspnetcore/security/authentication/identity-configuration.md

```diff
@@ -6,15 +6,15 @@ uid: security/authentication/identity-configuration
 
 ASP.NET Core Identity has some default behaviors that you can override easily in your application's startup class.
 
-## Password's policy
+## Passwords policy
 
-By default, Identity requires very secure passwords who have to contains uppercase character, lowercase character, digits and some others restrictions that you sometimes need to simplify. It's very simple to do that, all the configuration is accessible in the startup class of your application.
+By default, Identity requires very secure passwords who have to contain an uppercase character, lowercase character, digits and some other restrictions that you sometimes need to simplify. It's very simple to do that, all the configuration is accessible in the startup class of your application.
```

Comment:
By default, Identity requires that passwords contain an uppercase character, lowercase character, and digits. There are also some other restrictions.  If you want to simplify password restrictions, you can do that in the startup class of your application.